learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,85 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
+## Prerequisites
+
++ [Docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
++ You [log in](https://docs.docker.com/engine/reference/commandline/login/) your docker with [docker hub]((https://hub.docker.com/)) account
++ You have and know your **public IP** address
++ You have **a bitcoin and a litecoin address** for receiving mining reward
++ Your environment is open for incoming connections through port 2136.  
+
+
+## Installation
+
+### Step 1: Download and Run Script
+
+Downloaded script will help you to manage bitmark node container. The  installation of the script will pull down a latest bitmark node container image from docker hub and then start a bitmark node container for you.  To check what functions the script supports and how to run it, simply run ``` bash install-node-linux-mac.sh```.  or ```bitmarkNode-HyperV.bat```.
+
+
++  Linux and mac users
+    + [Download scripts for linux and mac](https://bitmark-node-docker-scripts.s3-ap-northeast-1.amazonaws.com/install-node-linux-mac.sh)
+    + Execute bash install-node-linux-mac.sh [Your Public IP]. When run the script without providing [Your Public IP], the script will find your IP from Internet
```

Comment:
In-line commands should usually be marked for clarity: `bash install-node-linux-mac.sh [Your Public IP]`

Reply:
If you're just going to list it in a full example (as you do here), then you can probably have something shorter in the description, e.g.:

* Execute the install script using `bash`, including your Public IP address. If you do not know your IP address, you can omit that, and the script will try to determine it from the internet:

        ```bash install-node-linux-mac.sh  <Your Public IP>```

Reply:
got it